docs/documentation.org

```diff
@@ -0,0 +1,96 @@
+#+STARTUP: indent
+#+TITLE: Macnica DHW
+* Macnica DHW Instructions for Rockchip MINIEVB RV1108 development kit
+** Getting the ~repo~ tool
+The first step is to download and prepare all the Yocto Project (YP)
+environment, you need to have the ~repo~ tool installed in your host machine.
+
+For Debian/Ubuntu Linux distro you can install it running:
+#+BEGIN_SRC shell
```

Comment:
```suggestion
#+STARTUP: indent
#+TITLE: Macnica DHW
* Supported boards:
** Rockchip MINIEVB RV1108 development kit
* Setup instructions
** Getting the ~repo~ tool
The first step is to download and prepare all the Yocto Project (YP)
environment, you need to have the ~repo~ tool installed in your host machine.

For Debian/Ubuntu Linux distro you can install it running:
#+BEGIN_SRC shell
```

Reply:
Before the `Getting the repo tool` we could add another section, with:

* Required Packages for the Build Host

The list of packages you need on the host development system can be large when covering all build scenarios using the Yocto Project. See [Yocto Project reference manual](https://www.yoctoproject.org/docs/3.1/ref-manual/ref-manual.html#detailed-supported-distros) for details.

Reply:
We might put a:
```
* Document revision

| Version | Description     | Release date |
|---------+-----------------+--------------|
|     1.0 | Initial release |   04-30-2020 |
|---------+-----------------+--------------|
```